libkbfs/prefetcher.go

```diff
@@ -122,32 +128,35 @@ func (p *blockPrefetcher) request(priority int, kmd KeyMetadata, ptr BlockPointe
 	}
 }
 
-func (p *blockPrefetcher) prefetchIndirectFileBlock(b *FileBlock, kmd KeyMetadata, priority int) {
+func (p *blockPrefetcher) prefetchIndirectFileBlock(b *FileBlock, kmd KeyMetadata) {
 	// Prefetch the first <n> indirect block pointers.
 	// TODO: do something smart with subsequent blocks.
 	numIPtrs := len(b.IPtrs)
 	if numIPtrs > defaultIndirectPointerPrefetchCount {
 		numIPtrs = defaultIndirectPointerPrefetchCount
 	}
+	p.log.CDebugf(context.Background(), "Prefetching pointers for indirect file block. Num pointers to prefetch: %d", numIPtrs)
 	for _, ptr := range b.IPtrs[:numIPtrs] {
 		p.request(fileIndirectBlockPrefetchPriority, kmd,
 			ptr.BlockPointer, b.NewEmpty(), "")
 	}
 }
 
-func (p *blockPrefetcher) prefetchIndirectDirBlock(b *DirBlock, kmd KeyMetadata, priority int) {
+func (p *blockPrefetcher) prefetchIndirectDirBlock(b *DirBlock, kmd KeyMetadata) {
 	// Prefetch the first <n> indirect block pointers.
 	numIPtrs := len(b.IPtrs)
 	if numIPtrs > defaultIndirectPointerPrefetchCount {
 		numIPtrs = defaultIndirectPointerPrefetchCount
 	}
+	p.log.CDebugf(context.Background(), "Prefetching pointers for indirect dir block. Num pointers to prefetch: %d", numIPtrs)
```

Comment:
Please use `context.TODO()` here and below.  We should probably make a new log debug tag to identify background prefetches, to make debugging easier.  Can be in a followup PR though.